src/components/table-transactions/table-transactions.scss

```diff
@@ -0,0 +1,50 @@
+.table-transactions {
+  border: 1px solid #ddd;
+  border-collapse: collapse;
+  width:100%;
```

Comment:
please run formatter through your code (and make sure that you have precommit hooks working)